qdrant-landing/content/documentation/examples/qdrant-airflow-astronomer.md

```diff
@@ -178,12 +178,12 @@ def recommend_book():
     ) -> None:
         hook = QdrantHook(conn_id=QDRANT_CONNECTION_ID)
 
-        result = hook.conn.search(
+        result = hook.conn.query_points(
             collection_name=COLLECTION_NAME,
-            query_vector=preference_embedding,
+            query=preference_embedding,
             limit=1,
             with_payload=True,
-        )
+        ).points
```

Comment:
Don't we need to specify the name of the vector to use (the `using` parameter)?

Reply:
The collection was created with the default vector.